stan/math/prim/prob/neg_binomial_2_lpmf.hpp

```diff
@@ -49,21 +58,17 @@ return_type_t<T_location, T_precision> neg_binomial_2_lpmf(
   size_t len_ep = max_size(mu, phi);
   size_t len_np = max_size(n, phi);
 
-  VectorBuilder<true, T_partials_return, T_location> mu__(size(mu));
-
-  for (size_t i = 0, max_size_seq_view = size(mu); i < max_size_seq_view; ++i) {
+  size_t len_mu = size(mu);
+  VectorBuilder<true, T_partials_return, T_location> mu__(len_mu);
+  for (size_t i = 0; i < len_mu; ++i) {
```

Comment:
I've been removing uses of trailing underscores for local variables. You could rename this to `mu_val`, for example.

Reply:
Thanks, will do.